lib/administrate/field/has_many.rb

```diff
@@ -11,8 +11,12 @@ def self.permitted_attribute(attr, _options = nil)
         { "#{attr.to_s.singularize}_ids".to_sym => [] }
       end
 
-      def associated_collection(order = self.order)
-        Administrate::Page::Collection.new(associated_dashboard, order: order)
+      def associated_collection(page = 1, order = self.order)
```

Comment:
I'm thinking that given that we are changing the method signature, we may as well change it to use keyword arguments. Positional arguments look a bit arbitrary to me. (I see it's the same as the old private `#resources`).